t/qi.lisp

```diff
@@ -75,14 +75,18 @@
 
 ;; Tests that even if the tarball doesn't have the same name as what we expect, we still
 ;; sucessfully unpack it and load it.
-(let* ((dep (qi::make-dependency :name "anaphora"
-                                 :location (qi.packages::http "git@github.com/tokenrove/anaphora")
-                                 :src-path (merge-pathnames tar-dir "anaphora-master.tar.gz")
-                                 :sys-path (merge-pathnames tar-dir "anaphora-latest")
-                                 :version "latest"))
-       (out (qi.packages::unpack-tar dep tar-dir)))
-  (ok (not (eql out (dependency-sys-path dep))) "Extracts tarball with different name.")
-  (ok (probe-file (dependency-sys-path dep)) "Extracts tarball and exists"))
+(let ((dep (qi::make-dependency :name "anaphora"
+                                :location (qi.packages::http "https://github.com/tokenrove/anaphora/tarball/master")
+                                :src-path (merge-pathnames tar-dir "anaphora-master.tar.gz")
+                                :sys-path (merge-pathnames tar-dir "anaphora-latest")
+                                :version "latest")))
+  (qi::bootstrap (qi.packages::dependency-name dep))
+  (adt:match location (dependency-location dep)
+             ((http url)
+              (qi.packages::download-tarball url dep)))
```

Comment:
Since the tarball cache was moved out of the Qi path and into a temporary directory, I just had it download the tarball instead of using a fixture.  Alternatively I guess we could just copy the fixture into `/tmp/qi`.